EFAuditer/Auditer.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Audit.Core;
+using Newtonsoft.Json;
+
+namespace EFAuditer
+{
+    public static class Auditer
+    {
+        // TODO NTBS-218: Split this into a static class just containing the setup method to be called once, and a non-static auditer that can be injected as a singleton into the application that uses this

```

Comment:
I've got everything in this static class for now, but I don't think it's good practice - it's ok for the setup method as that only gets called once, but it probably makes more sense to dependency inject an Auditer class as a singleton into the controllers that need it and then call the method on that. There has been some refactoring of the ValidationBaseModel/NotificationBaseModel, so I thought this might be easier to do once I've merged into the changes on master.